File: Python-Assignment/Corn-flakes/passScore.py
passes = 0
failures = 0

for index in range(0, 15):
	score = int(input("Enter a score: "))

	if score >= 45:
		passes = passes + 1
	else:
		failures = failures + 1

print(passes, "students passed the examination")
print(failures, "students failed the examination")

File: Python-Assignment/cornFlakes/PassScore.java
import java.util.Scanner;

public class PassScore{
   public static void main(String []args){

	Scanner input = new Scanner(System.in);

	int passes = 0;
	int failures = 0;

	for(int index = 1; index <= 15; index++){
		System.out.print("Enter a score: ");
		int score = input.nextInt();

		if (score >= 45)
			passes++;
		else
			failures++;
	}

	System.out.println(passes + " students passed the examination");
	System.out.println(failures + " students failed the examination");
   }
}